Us004 load events

madeye-matt opened the pull request:
Added a loader script for loading collection exercise events and added test data received from Lou

madeye-matt committed:
Updated cfdatabasetool Dockerfile to allow Java debugging

madeye-matt committed:
Refactored loader ahead of starting event loader

madeye-matt committed:
Loads events to collection exercise service

but makes no attempt to check the date format

madeye-matt committed:
Finished loader

Dates now sent up in ISO9601 format

andrewmil reviewed (changes requested):
We should probably also update `collex-loader/Readme.md` It's not totally obvious how to run it/if it needs to be run separately/together with `load.py`

madeye-matt committed:
Nasty ISO8601 date hack

Hacked load.py to remove the last colon from the date string as passed
to the collection exercise service (Jackson by default only accepts
timezone with no colon).  The collection exercise service can be changed
to use/accept flexible ISO dates but then it sends dates to OTHER Java
services that won't accept the flexible dates (basically ALL the Java
services will need to be updated)

madeye-matt committed:
Updated README to reflect multiple scripts

madeye-matt committed:
Fixed script name

madeye-matt commented:
@andrewmil README.md updated as requested

andrewmil commented on collex-loader/load_events.py:
```diff
@@ -0,0 +1,102 @@
+        raise
+
+    time_str = raw.isoformat(timespec='milliseconds')
+    // WATCH OUT: this removes the last : from a valid ISO8601 date string to form another
```
Wrong way to comment in python

madeye-matt replied:
Ooops! *looks shifty* fixed now

tej99 reviewed (approved):
LG 👍

madeye-matt committed:
Fixed ahem dodgy java comments in python

benjefferies commented:
We will need to update the Makefile in https://github.com/ONSdigital/ras-integration-tests too
